gui/system/migrations/0047_auto_20200527_1038.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should we change the name of this migration to what this is doing like all the other migration files?